src/zcl_abapgit_repo_online.clas.abap

```diff
@@ -337,7 +337,7 @@ CLASS ZCL_ABAPGIT_REPO_ONLINE IMPLEMENTATION.
 
   METHOD select_commit.
     reset_remote( ).
-    mv_current_commit = iv_sha1.
+    set( iv_selected_commit = iv_selected_commit ).
```

Comment:
this should also reset the branch name, so the code can know its now a commie SHA1 that the user has chosen?

Reply:
...and vice versa.

Reply:
yes